src/components/ReviewRow/index.tsx

```diff
@@ -9,7 +9,11 @@ import './index.scss';
  */
 const ReviewRow = ({ children, className }: any) => {
   const classes = classnames('easi-review-row', className);
-  return <div className={classes}>{children}</div>;
+  return (
+    <div className={classes} data-testid="review-row">
```

Comment:
Same as above, since `ReviewRow` is rendered so many times. all the test is ensuring is that the `dt` and `dd` are inside a `div`, which isn't a big deal.